src/core/cgroup.c

```diff
@@ -3510,6 +3510,35 @@ void manager_invalidate_startup_units(Manager *m) {
                 unit_invalidate_cgroup(u, CGROUP_MASK_CPU|CGROUP_MASK_IO|CGROUP_MASK_BLKIO);
 }
 
+int compare_job_cpu_weight(const void *a, const void *b)
+{
+        Job *x = (Job*)a, *y = (Job*)b;
```

Comment:
If `const` is added, the cast can dropped, no?

Reply:
Right, of course. I was in a C++ mindset where the `const` usually propagates and the unit cannot be `const` below. But that's not a problem here.